src/dymaptic.GeoBlazor.Core/Scripts/arcGisJsInterop.ts

```diff
@@ -2155,6 +2157,14 @@ async function createWidget(widget: any, viewId: string): Promise<Widget | null>
                 icon: widget.icon ?? undefined,
             });
             break;
+        case 'areaMeasurement2D':
+            newWidget = new AreaMeasurement2D({
+                view: view,
+                viewModel: widget.AreaMeasurement2DViewModel ?? undefined,
+                unit: widget.unit ?? undefined,
+                unitOptions: widget.unitOptions ?? undefined
+            });
```

Comment:
Can you add unit tests that test with/without the unit and options? I realize we are probably waiting on the ViewModel as we have for other types.

Reply:
in an earlier commit, i did get start building out the viewmodel.  but decided against it going through it, since we have those other ouststanding viewmodels to do also.  I think once I complete this PR.  I think we can add this ViewModel in another PR and start to knock out those other 4... so we can start some to add some customization